actionsystemtest/lib/action_system_test/driver_adapters.rb

```diff
@@ -0,0 +1,46 @@
+module ActionSystemTest
+  # == System Testing Driver Adapters
+  #
+  # By default Rails supports Capybara with the Selenium Driver. Rails provides
+  # configuration setup for using the selenium driver with Capybara.
+  # Additionally Rails can be used as a layer between Capybara and its other
+  # supported drivers: +:rack_test+, +:selenium+, +:webkit+, or +:poltergeist+.
+  #
+  # *{RackTest}[https://github.com/jnicklas/capybara#racktest]
+  # *{Selenium}[http://seleniumhq.org/docs/01_introducing_selenium.html#selenium-2-aka-selenium-webdriver]
+  # *{Webkit}[https://github.com/thoughtbot/capybara-webkit]
+  # *{Poltergeist}[https://github.com/teampoltergeist/poltergeist]
+  #
+  # === Driver Features
+  #
+  # |                 | Default Browser       | Supports Screenshots? |
+  # |-----------------|-----------------------|-----------------------|
```

Comment:
Nitpick: saw another table where we put spaces between the rows and pipes, e.g.: `---- | ----`. Not sure if we have a preferred style.